schedule/yam/agama_unattended_medium.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
`unattended` always as postfix please. Instead of 'medium" which doesn't mean anything because there are 3 mediums currently (from_medium perhaps is what you want to express). we could use something that will match the test suite which in turn should have a 'nice' functional name, for example:
`agama_post_registration.yaml` and name of the the test suite `sles_post_registration`.

Reply:
Renamed the file name, thanks.